invenio_records_rest/views.py

```diff
@@ -64,42 +64,39 @@ def elasticsearch_query_parsing_exception_handler(error):
     return InvalidQueryRESTError(description=description).get_response()
 
 
-elasticsearch_error_handlers = {
```

Comment:
You can change this variable to be a proxy that resolves the `RECORDS_REST_ELASTICSEARCH_ERROR_HANDLERS ` configuration option without changing the API of `create_error_handlers`.